patches/chrome-browser-ui-BUILD.gn.patch

```diff
@@ -22,15 +22,23 @@ index eae7b4cef4dc9e633c975b63fcf89ebe59dffff6..9218dbd11b83acaad5274943b3762c66
      }
    }
  
-@@ -5489,6 +5493,7 @@ static_library("ui_public_dependencies") {
+@@ -5478,6 +5482,7 @@ static_library("ui_public_dependencies") {
+     "//components/enterprise/buildflags",
+     "//components/paint_preview/buildflags",
+     "//components/safe_browsing:buildflags",
++    "//components/saved_tab_groups/public",
```

Comment:
I believe the right fix is to make `//components/saved_tab_groups/proto` a public dependency of `//components/saved_tab_groups/public`.